index.d.ts

```diff
@@ -679,7 +680,7 @@ declare module 'fastest-validator' {
 	/**
 	 * Description of validation rule definition for a some property
 	 */
-	type ValidationRule = ValidationRuleObject | ValidationRuleObject[] | ValidationRuleName | string;
+	type ValidationRule = ValidationRuleObject | ValidationRuleObject[] | ValidationRuleName | number;
```

Comment:
Why `number`?

Reply:
Nice catch. I've tested negative cases for `v.compile(1)` to throw errors

Reply:
Travis failed :(

Reply:
Yeah. Some last time moment fixes :(

Reply:
Now it's green